pyluba/aliyun/cloud_gateway.py

```diff
@@ -50,15 +50,19 @@ class CloudIOTGateway:
     _iotCredentials = {}
     converter = DatatypeConverter()
 
+    _regions_file_path = 'regions.json'
+    _iotCredentials_file_path = 'iotCredentials.json'
+    _mqtt_credentials_file_path = '_mqtt_credentials.json'
+
     def __init__(self):
         self._region = {}
         self._app_key = "34231230"
```

Comment:
add a const.py file in the same directory and add these two as consts
e.g

`APP_KEY = '123456'`